crates/bevy_reflect/src/path/mod.rs

```diff
@@ -271,22 +275,32 @@ pub trait GetPath: Reflect {
 // Implement `GetPath` for `dyn Reflect`
 impl<T: Reflect + ?Sized> GetPath for T {}
 
+/// An [`Access`] combined with an `offset` for extra context.
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, Hash)]
+pub struct OffsetAccess {
```

Comment:
It's not clear from reading these docs how a user might construct or read this public type.

Reply:
I made the struct's fields public and added a few `From<_>` implementations to make stuff easier.